cnf-tests/oslat-run.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# env vars:
+# RTPRIO (default to 1): set the SCHED_FIFO priority
+# RUNTIME_SECONDS (default 10 minute): how long the test will run
+# LOG_DIR (default stdout): the log directory where to save the ouput of the oslat command
+
+RTPRIO=${RTPRIO:-1}
+RUNTIME_SECONDS=${RUNTIME_SECONDS:-600}
+
+cpulist=()
+for cpu in $(cat /proc/self/status | grep Cpus_allowed_list: | cut -f 2 | awk '/-/{for (i=$1; i<=$2; i++)printf "%s%s",i,ORS;next} 1' RS=, FS=-)
+do
+	cpulist+=(${cpu})
+done
+
+echo "############# dumping env ###########"
+env
+echo "#####################################"
+
+echo " "
+echo "########## container info ###########"
+echo "/proc/cmdline: $(cat /proc/cmdline)"
+echo "allowed cpu list: ${cpulist[@]}"
+echo "uname -nr: $(uname -nr)"
+echo "#####################################"
+
+main_thread_cpu="${cpulist[0]}"
```

Comment:
not sure if this is guaranteed

Reply:
under the test, I have some assumptions, one of them that the test container requester >= 2 CPUs, so the first CPU will be used for the main thread and the second for the latency test.

Reply:
fair enough, but please add a comment to make this assumption explicit.